app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -44,22 +42,18 @@ import scala.concurrent.Future
 import scala.util.{Failure, Success}
 
 @Singleton
-class RegisterController @Inject() (val messagesApi:             MessagesApi,
-                                    val helpToSaveService:       HelpToSaveService,
-                                    val sessionCacheConnector:   SessionCacheConnector,
-                                    frontendAuthConnector:       FrontendAuthConnector,
-                                    jsonSchemaValidationService: JSONSchemaValidationService,
-                                    metrics:                     Metrics,
-                                    app:                         Application,
-                                    pagerDutyAlerting:           PagerDutyAlerting,
-                                    configuration:               Configuration
+class RegisterController @Inject() (val messagesApi:           MessagesApi,
+                                    val helpToSaveService:     HelpToSaveService,
+                                    val sessionCacheConnector: SessionCacheConnector,
+                                    frontendAuthConnector:     FrontendAuthConnector,
+                                    metrics:                   Metrics,
+                                    app:                       Application,
+                                    pagerDutyAlerting:         PagerDutyAlerting,
```

Comment:
i guess we dont need `pagerDutyAlerting` here anymore ?

Reply:
no we don't need it - I'll remove it